From 009bb3c5930ce1bdffa2e3bd003f46158e303d71 Mon Sep 17 00:00:00 2001
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Tue, 28 Sep 2010 12:00:28 -0700
Subject: [PATCH 0214/1092] cbus: move EXPORT_SYMBOL close to its exported
 symbols

while there, also add a missing static to cbus_bus_init().

Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 drivers/cbus/cbus.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/drivers/cbus/cbus.c b/drivers/cbus/cbus.c
index 0bcc211..d73d0c9 100644
--- a/drivers/cbus/cbus.c
+++ b/drivers/cbus/cbus.c
@@ -38,6 +38,7 @@
 #include "cbus.h"
 
 struct cbus_host *cbus_host = NULL;
+EXPORT_SYMBOL(cbus_host);
 
 #ifdef CONFIG_ARCH_OMAP1
 /* We use our own MPUIO functions to get closer to 1MHz bus speed */
@@ -210,6 +211,7 @@ int cbus_read_reg(struct cbus_host *host, int dev, int reg)
 {
 	return cbus_host ? cbus_transfer(host, dev, reg, -1) : -ENODEV;
 }
+EXPORT_SYMBOL(cbus_read_reg);
 
 /*
  * Write to a given register of the device
@@ -218,8 +220,9 @@ int cbus_write_reg(struct cbus_host *host, int dev, int reg, u16 val)
 {
 	return cbus_host ? cbus_transfer(host, dev, reg, (int)val) : -ENODEV;
 }
+EXPORT_SYMBOL(cbus_write_reg);
 
-int __init cbus_bus_init(void)
+static int __init cbus_bus_init(void)
 {
 	struct cbus_host *chost;
 	int ret;
@@ -286,10 +289,6 @@ exit1:
 
 subsys_initcall(cbus_bus_init);
 
-EXPORT_SYMBOL(cbus_host);
-EXPORT_SYMBOL(cbus_read_reg);
-EXPORT_SYMBOL(cbus_write_reg);
-
 MODULE_DESCRIPTION("CBUS serial protocol");
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("Juha Yrjölä, David Weinehall, and Mikko Ylinen");
-- 
1.7.11.2

